huanyu-li/week5/mon/games_on_rails/app/views/magic8ball/home.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<h3>Magic 8 Ball</h3>
+
+<p>How to use:</p>
+
+<p>Enter your question at the URL bar then the magic 8 ball will answer you</p>
+
+<p>i.e. <code><a href="http://localhost:3000/magic-8-ball/Why do I have to be harassed into submitting homework?">http://localhost:3000/magic-8-ball/Why do I have to be harassed into submitting homework?</a></code></p>
+
```

Comment:
You don't really want the user to enter in a question through the URL. 

You can create a form and then get the question from params.

Reply:
Hi Kris, 
Thanks for pointing it out.  😂 I have the same question when I read the 'description' on [https://gist.github.com/textchimp/378e0b7fff224fffe56402e7159f52c3](url) .  I thought it asks us to do this on purpose 😂. Actually, I wrote this in a form when I was doing this homework, then I read the 'description' again, and changed it to URL version just before I submitted it.